pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -128622,10 +128622,9 @@ self: {
        hydraPlatforms = stdenv.lib.platforms.none;
        broken = true;
      }) {GraphicsMagick = null; inherit (pkgs) bzip2; freetype2 = null; 
-         inherit (pkgs) jasper; inherit (pkgs) lcms; 
-         inherit (pkgs) libjpeg; inherit (pkgs) libpng; 
-         inherit (pkgs) libxml2; tiff = null; wmflite = null; 
-         inherit (pkgs) zlib;};
+         jasper = null; inherit (pkgs) lcms; inherit (pkgs) libjpeg; 
+         inherit (pkgs) libpng; inherit (pkgs) libxml2; tiff = null; 
+         wmflite = null; inherit (pkgs) zlib;};
```

Comment:
This file is automatically generated by `hackage2nix` (and `cabal2nix`).

Even if you fix this, it will still be regenerated just like the above next time it is regenerated.

I'd suggest just removing this change for now.  The Haskell package this is modifying is already marked broken.

Reply:
I regenerated this file using hackage2nix, in a separate commit from the main removal. So I think this should be fine? I'm just reducing the window where hackage-packages.nix is inconsistent with the rest of nixpkgs, but the next time someone generates this they should get exactly the same output.

Reply:
@danderson In general we only change the `pkgs/development/haskell-modules/hackage-packages.nix` file in the `haskell-updates` branch, in order to avoid any merge conflicts when merging `haskell-updates` into `master`.

I'll defer to @peti as to whether or not this is okay to do here in this PR.

Reply:
Removed the haskell-packages.nix change. Can easily put it back if preferred.